experimental/handoff-library/csharp_dotnetcore/samples/LPProxyBot/LiverPersonConnector/LivePersonConnector.cs

```diff
@@ -0,0 +1,279 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace LivePersonConnector
+{
+    public interface ICredentialsProvider
+    {
+        string LpAccount { get; }
+        string LpAppId { get; }
+        string LpAppSecret { get; }
+        string MsAppId { get; }
+    }
+
+    internal class LivePersonConversationRecord
+    {
+        public string ConversationId;
+        public string MsgDomain;
+        public string AppJWT;
+        public string ConsumerJWS;
```

Comment:
What is a JWS? Maybe some comments to explain what the fields in this class mean? that is, if users need to understand.

Reply:
added comment